src/constants/constants.js

```diff
@@ -586,3 +588,20 @@ export const BLOCK_TIME_MILLIS = {
 };
 
 export const TRANSACTION_CONFIRMATIONS = 1;
+
+export const CACHE_DATA_TYPES = {
+    NUMBER: 'number',
+};
+
+export const CACHED_FUNCTIONS = {
+    ParametersStorage: [
```

Comment:
Let's make it an object, with `parameterName` as key and `type` as value, then we don't need to iterate through all parameters every time.

If we remove types as I suggested in the comment above, we can just make it a `Set` with constant `O(1)` lookup